tests/cbmc/include/cbmc_proof/cbmc_utils.h

```diff
@@ -28,6 +30,27 @@ struct store_byte_from_buffer {
     uint8_t byte;
 };
 
+/**
+ * Asserts whether s2n_stuffer's fields are equivalent, except for read_cursor,
+ * which might change after a read operation.
+ */
+ void assert_stuffer_immutable_fields_after_read(const struct s2n_stuffer *lhs,
+                                                 const struct s2n_stuffer *rhs,
+                                                 const struct store_byte_from_buffer *rhs_byte);
+/**
+ * Asserts whether s2n_blob objects are equivalent.
+ */
+ void assert_blob_equivalence(const struct s2n_blob *lhs,
```

Comment:
The semantics of the RHS seems a little funny - you should add a comment clarifying.

Reply:
I updated the variable name and the documentation.